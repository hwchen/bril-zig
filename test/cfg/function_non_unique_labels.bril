@main() {
    v: int = const 2;
    call @fn1;
    call @fn2;
    print v;
}
@fn1 {
    v: int = const 2;
    jmp .testlabel;
.testlabel:
    print v;
    ret;
}
@fn2 {
    v: int = const 4;
    jmp .testlabel;
.testlabel:
    print v;
    ret;
}
