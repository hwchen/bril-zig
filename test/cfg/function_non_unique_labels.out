{"0":["1","3"],"1":["testlabel"],"testlabel":[],"3":["testlabel"]}
